CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Global owners
+
+* @balopat @dgageot @nkubala @priyawadhwa @tejal29
+
+# JIB owners
+
+**/**jib** @GoogleContainerTools/java-tools
```

Comment:
maybe @GoogleContainerTools/java-tools-build ? I would prefer the tighter ownership scope
https://github.com/orgs/GoogleContainerTools/teams/java-tools-build

Reply:
we could do that - or make it explicitly list owners - that way any change to this is driven by this file instead of group admins - giving more transparency. WDYT?

Reply:
I'm okay with that. It's only marginally more work to add/remove users from our team.